test/recipes/95-test_external_krb5_data/krb5.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh -ex
+#
+# Copyright 2017 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the OpenSSL license (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+# krb5's test suite clears LD_LIBRARY_PATH
+LDFLAGS="-L`pwd`/$BLDTOP -Wl,-rpath,`pwd`/$BLDTOP"
```

Comment:
For some reason I am remembering a discussion of new-dtags vs. -rpath, but didn't see it in a skim of the overview page.
In any case, I guess it doesn't really matter here, since this is just for running the test suite.